api/src/main/java/org/pmiops/workbench/mail/MailServiceImpl.java

```diff
@@ -86,40 +82,38 @@ public void sendBetaAccessRequestEmail(final String userName) throws MessagingEx
   public void sendWelcomeEmail(final String contactEmail, final String password, final User user)
       throws MessagingException {
 
-    final MandrillMessage msg =
```

Comment:
Most sendEmail methods have a block like this.  Push this boilerplate into a new version of `sendWithRetries()`

Reply:
Please confirm that the parameters of these methods have not changed in your review.